ansible/roles/ocp4-workload-openbanking-sandbox/tasks/post_workload.yml

```diff
@@ -0,0 +1,335 @@
+---
+# Implement your Post Workload deployment tasks here
+
+###
+# Keycloak / Microcks tasks
+
+# - name: Get route to Keycloak/Microcks service
+#   k8s_info:
+#     api_version: route.openshift.io/v1
+#     kind: Route
+#     name: "{{ obsandbox_microcks_keycloak_route }}"
+#     namespace: "{{ obsandbox_microcks_namespace }}"
+#   register: _keycloak_route
+#   retries: 3
+#   delay: 5
+#   until: _keycloak_route.resources|length > 0
+
+# - set_fact:
+#     _keycloak_url: "https://{{ _keycloak_route.resources[0].spec.host }}"
+
+- name: Retrieve the Keycloak/Microcks admin credential in the Secret
+  k8s_info:
+    api_version: v1
+    kind: Secret
+    name: "{{ obsandbox_microcks_keycloak_secret }}"
+    namespace: "{{ obsandbox_microcks_namespace }}"
+  register: _microcks_keycloak_secret
+  retries: 3
+  delay: 5
+  until: _microcks_keycloak_secret.resources|length > 0
+
+- set_fact:
+    _keycloak_admin_usr: "{{ _microcks_keycloak_secret.resources[0].data.username | b64decode }}"
+    _keycloak_admin_pwd: "{{ _microcks_keycloak_secret.resources[0].data.password | b64decode }}"
+
+# - name: Authenticating Keycloak user "{{ _keycloak_admin_usr }}" via OAuth2
```

Comment:
Disabled code blocks should not be added to version control.

Reply:
code block with comments removed